scripts/local-dev.sh

```diff
@@ -221,33 +224,59 @@ start_local_services() {
     export LLAMA_STACK_URL="http://localhost:$LLAMASTACK_PORT/v1/openai/v1"
     export THANOS_TOKEN="$TOKEN"
     export METRICS_API_URL="http://localhost:$METRICS_API_PORT"
+    export MCP_URL="http://localhost:$MCP_PORT"
     export PROM_URL="http://localhost:$THANOS_PORT"
     
     # macOS weasyprint support
     export DYLD_FALLBACK_LIBRARY_PATH="/opt/homebrew/lib:$DYLD_FALLBACK_LIBRARY_PATH"
 
     set_model_config
     
-    # Start MCP service
+    # Start Metrics API backend
     echo -e "${BLUE}🔧 Starting Metrics API backend...${NC}"
     ensure_port_free "$METRICS_API_PORT"
     (cd src/api && python3 -m uvicorn metrics_api:app --host 0.0.0.0 --port $METRICS_API_PORT --reload > log.txt) &
     MCP_PID=$!
     
-    # Wait for MCP to start
+    # Wait for Metrics API to start
     sleep 3
     
-    # Test MCP service
+    # Test Metrics API service
     if curl -s --connect-timeout 5 "http://localhost:$METRICS_API_PORT/models" > /dev/null; then
         echo -e "${GREEN}✅ Metrics API backend started successfully${NC}"
     else
         echo -e "${RED}❌ Metrics API backend failed to start${NC}"
         exit 1
     fi
+
+    # Start MCP server (HTTP transport)
+    echo -e "${BLUE}🧩 Starting MCP Server (HTTP)...${NC}"
+    ensure_port_free "$MCP_PORT"
+    (cd src && \
+      MCP_TRANSPORT_PROTOCOL=http \
+      MODEL_CONFIG="$MODEL_CONFIG" \
+      PROMETHEUS_URL="$PROMETHEUS_URL" \
+      LLAMA_STACK_URL="$LLAMA_STACK_URL" \
+      THANOS_TOKEN="$THANOS_TOKEN" \
+      python3 -m mcp_server.main > mcp_log.txt) &
+    MCP_SRV_PID=$!
+
+    # Wait for MCP server to start
+    sleep 3
+
+    # Test MCP server health
+    if curl -s --connect-timeout 5 "http://localhost:$MCP_PORT/health" | grep -q '"status"'; then
+        echo -e "${GREEN}✅ MCP Server started successfully on port $MCP_PORT${NC}"
+    else
+        echo -e "${RED}❌ MCP Server failed to start${NC}"
+        exit 1
+    fi
     
     # Start Streamlit UI
     echo -e "${BLUE}🎨 Starting Streamlit UI...${NC}"
-    (cd src/ui && streamlit run ui.py --server.port $UI_PORT --server.address 0.0.0.0 --server.headless true) &
+    (cd src/ui && \
+      MCP_SERVER_URL="http://localhost:$MCP_PORT" \
```

Comment:
Why exposing MCP_SERVER_URL is needed here while METRICS_API_URL isn't?

Reply:
This is making  MCP_PORT configurable here instead of changing in the code, we can add that too for API, I will cover that in next PR if it's fine.